bullet/src/main/java/org/bytedeco/bullet/presets/BulletCollision.java

```diff
@@ -0,0 +1,152 @@
+                "BulletCollision/CollisionShapes/btTriangleMesh.h",
+                "BulletCollision/CollisionShapes/btConvexTriangleMeshShape.h",
+                "BulletCollision/CollisionShapes/btTriangleMeshShape.h",
+                "BulletCollision/CollisionShapes/btOptimizedBvh.h",
+                "BulletCollision/CollisionShapes/btTriangleInfoMap.h",
+                "BulletCollision/CollisionShapes/btBvhTriangleMeshShape.h",
+                "BulletCollision/CollisionShapes/btScaledBvhTriangleMeshShape.h",
+                "BulletCollision/CollisionShapes/btCompoundShape.h",
+                "BulletCollision/CollisionShapes/btTetrahedronShape.h",
+                "BulletCollision/CollisionShapes/btEmptyShape.h",
+                "BulletCollision/CollisionShapes/btMultiSphereShape.h",
+                "BulletCollision/CollisionShapes/btUniformScalingShape.h",
+            },
+            link = "BulletCollision"
+        )
+    },
+    target = "org.bytedeco.bullet.BulletCollision",
+    global = "org.bytedeco.bullet.global.BulletCollision"
+)
+public class BulletCollision implements InfoMapper {
+    static { Loader.checkVersion("org.bytedeco", "bullet"); }
+
+    public void map(InfoMap infoMap) {
+        infoMap
+            .put(new Info("bt32BitAxisSweep3").base("btBroadphaseInterface"))
+            .put(new Info("btAxisSweep3").base("btBroadphaseInterface"))
+
+            .put(new Info("BT_DECLARE_STACK_ONLY_OBJECT").cppText("#define BT_DECLARE_STACK_ONLY_OBJECT"))
+            .put(new Info("btCollisionObjectData").cppText("#define btCollisionObjectData btCollisionObjectFloatData"))
+            .put(new Info("btOptimizedBvhNodeData").cppText("#define btOptimizedBvhNodeData btOptimizedBvhNodeFloatData"))
+            .put(new Info("btPersistentManifoldData").cppText("#define btPersistentManifoldData btPersistentManifoldFloatData"))
+            .put(new Info("btQuantizedBvhData").cppText("#define btQuantizedBvhData btQuantizedBvhFloatData"))
+
+            .put(new Info("DBVT_INLINE").cppTypes().annotations())
+
+            .put(new Info(
+                    "DBVT_BP_PROFILE",
+                    "DEBUG_PERSISTENCY",
+                    "NO_VIRTUAL_INTERFACE",
+                    "PFX_USE_FREE_VECTORMATH",
+                    "__SPU__"
+                ).define(false))
+
+            .put(new Info("btAlignedObjectArray<btBvhSubtreeInfo>").pointerTypes("btAlignedObjectArray_btBvhSubtreeInfo"))
+            .put(new Info("btAlignedObjectArray<btCollisionObject*>").pointerTypes("btAlignedObjectArray_btCollisionObjectPointer"))
+            .put(new Info("btAlignedObjectArray<btIndexedMesh>").pointerTypes("btAlignedObjectArray_btIndexedMesh"))
+            .put(new Info("btAlignedObjectArray<btPersistentManifold*>").pointerTypes("btAlignedObjectArray_btPersistentManifoldPointer"))
+            .put(new Info("btAlignedObjectArray<btQuantizedBvhNode>").pointerTypes("btAlignedObjectArray_btQuantizedBvhNode"))
```

Comment:
We don't need to map all the characters from the C++ type name. Here, for example, since it's obvious that btAlignedObjectArray is some sort of alternative to std::vector, we can name those instances more simply btBvhSubtreeInfoArray, btCollisionObjectArray (since there isn't another btCollisionObjectArray, I don't think we need "Pointer" in there), btIndexedMeshArray, etc.